### DISCUSSION TITLE
TRV window_open

### DISCUSSION THREAD

**Discussion by gu-stera** [windkh/node-red-contrib-shelly#123] [Q&A] [answered]
Hello,

How can I control window_open? I would like to make a flow if window or a door is opened to close the TRV valve.

thanks

**Comment by windkh** [comment, marked as answer]
@gu-stera just have a look at the exampe flow for the TRV. Here you can see how to turn the valve on and off.
https://github.com/windkh/node-red-contrib-shelly/blob/master/examples/trv.json

Just take a door/window node, add a switch node after it to check if the window is opened and then trigger the trv.
